service/src/enclave/tls_ra.rs

```diff
@@ -25,6 +25,7 @@ use sgx_types::*;
 use std::{
 	net::{TcpListener, TcpStream},
 	os::unix::io::AsRawFd,
+	sync::{atomic::AtomicBool, Arc},
```

Comment:
Seems a bit weird to me that we only have imports here.

Reply:
good catch. don't know what happened there